packages/gatsby-theme-docs/src/components/top-menu.js

```diff
@@ -169,7 +177,7 @@ const TopMenu = () => {
           event.stopPropagation();
         }}
       >
-        <ContentGrid>
```

Comment:
This grid component should stay, as all the children are positioned accordingly to the grid. If you change the container to a block element, the children will be break the layout most likely.

Reply:
@emmenko I just inspected the built site: The content has its own grid in the "Columns" component.  

This "ContentGrid" is an extra wrapper to position it the same way the main page content is positioned.